src/cli/commands/snip.ts

```diff
@@ -172,6 +195,18 @@ export const snip = async (args: SnipArgs): Promise<string[]> => {
     errors.push(message);
   }
 
+  // if an id was not used, print a warning
+  {
+    const idSet = typeof id === "string" ? new Set([id]) : new Set(id);
+    if (id && idsUsed.size !== idSet.size) {
+      const unused = Array.from(idSet).filter((x) => !idsUsed.has(x));
```

Comment:
Why are you creating a set just to turn it back into an array later? Maybe add a comment above saying something like "Eliminate duplicate id entries" if that's the intention here.

Reply:
renamed variable to `dedupIds` and added a comment. Agree it is not the best way to do these checks, but I think it is good enough for what this snippet of code is meant to accomplish.

Reply:
Thanks!